src/app/bot-dashboard/bot-dashboard.component.html

```diff
@@ -25,7 +25,7 @@
                                                 </div>
                                                 <div class="card-body text-left">
                                                     <p class="card-text mb-2"><b>280 Chats</b></p>
-                                                    <p class="font12 mb-2 color848c98">Last updated: Yesterday 8:12 AM</p>
+                                                    <p class="font12 mb-2 color848c98">Last updated: {{ getFormattedDate(botConfig.updatedTimestamp) }}</p>
```

Comment:
Add '?' in case updatedTimeStamp is not available.

Reply:
updated